tests/components/mqtt/test_init.py

```diff
@@ -1402,11 +1402,15 @@ async def test_setup_manual_mqtt_empty_platform(hass, caplog):
     assert "voluptuous.error.MultipleInvalid" not in caplog.text
 
 
+@patch("homeassistant.components.mqtt.PLATFORMS", [])
 async def test_setup_mqtt_client_protocol(hass):
     """Test MQTT client protocol setup."""
     entry = MockConfigEntry(
         domain=mqtt.DOMAIN,
-        data={mqtt.CONF_BROKER: "test-broker", mqtt.config.CONF_PROTOCOL: "3.1"},
+        data={
+            mqtt.CONF_BROKER: "test-broker",
+            mqtt.config_integration.CONF_PROTOCOL: "3.1",
```

Comment:
The constant was imported from `mqtt.config` this was moved to `mqtt.config_integration`